python-package/lightgbm/basic.py

```diff
@@ -1145,7 +1188,7 @@ def __init__(self, data, label=None, reference=None,
             sum(group) = n_samples.
             For example, if you have a 100-document dataset with ``group = [10, 20, 40, 10, 10, 10]``, that means that you have 6 groups,
             where the first 10 records are in the first group, records 11-30 are in the second group, records 31-70 are in the third group, etc.
-        init_score : list, numpy 1-D array, pandas Series or None, optional (default=None)
+        init_score : list, list of lists (for multi-class task), numpy array, pandas Series, pandas DataFrame (for multi-class task) or None, optional (default=None)
```

Comment:
@StrikerRUS do you think this should have a comma before `or None`? I did include it in the dask docstrings but I just realized this doesn't have it. I'll make them consistent but would like to know what you think is the correct one.

Reply:
I like how you did it. I believe a comma before `or None` prevents users from thinking that it possible to include `None`s into a list: https://github.com/microsoft/LightGBM/pull/4557#issue-720040131. However, I'm not sure whether it is grammatically correct or not. @jameslamb should know for sure 🙂

Reply:
I have a personally prefer `a, b, c, or None, optional (default=None) (with the `,` before `or None`),  but both are equally valid and I don't think you need to change anything